internal/pkg/policy/self.go

```diff
@@ -187,6 +222,49 @@ func (m *selfMonitorT) groupByLatest(policies []model.Policy) map[string]model.P
 	return groupByLatest(policies)
 }
 
+// getPolicyOutput will return the Elasticsearch output block of m.policy if there is a new revision.
+func (m *selfMonitorT) getPolicyOutput() (config.Elasticsearch, error) {
+	var policyES config.Elasticsearch
+	// policy revision has not changed
+	if m.policy.RevisionIdx == m.lastRev {
+		return policyES, ErrNoPolicyUpdate
```

Comment:
nit: it's not really an error if the policy hasn't changed, the function could return an extra boolean e.g. `hasChanged` to separate this scenario from errors